src/categories/components/CategoryProducts/CategoryProducts.tsx

```diff
@@ -28,34 +28,36 @@ export const CategoryProducts = ({
   onSelectProductsIds,
 }: CategoryProductsProps) => (
   <DashboardCard>
-    <DashboardCard.Title>
-      <Box display="flex" justifyContent="space-between" alignItems="center">
-        <FormattedMessage
-          id="+43JV5"
-          defaultMessage="Products in {categoryName}"
-          description="header"
-          values={{ categoryName: category?.name }}
-        />
+    <DashboardCard.Header>
+      <DashboardCard.Title>
```

Comment:
suggestion: `DashboardCard.Title` should contains only title, rest like the buttons should be in `DashboardCard.Header`